samples/spring-security-xsuaa-usage/xs-security.json

```diff
@@ -1,23 +1,47 @@
 {
 	"xsappname": "spring-security-xsuaa-usage",
 	"tenant-mode": "dedicated",
-	"scopes": 
-	[
-		{
-			"name": "$XSAPPNAME.Display",
-			"description": "With this scope, information can be read."
-		}
-	],
-
-	"role-templates": 
-	[
-		{
-			"name": "AccessRole",
-			"description": "Role to call the  sap-java-buildpack-api-usage service",
-			"scope-references": 
-			[
-				"$XSAPPNAME.Display"
-			]
-		}
-	]
+	"scopes"        : [
+    {
+      "name"        : "$XSAPPNAME.Read",
+      "description" : "Read Permissions."
+    },
+    {
+      "name"        : "$XSAPPNAME.Write",
+      "description" : "Write permissions."
+    }
+  ],
+  "role-templates": [
+    {
+      "name"             : "Viewer",
+      "description"      : "View Data",
+      "scope-references" : [
+        "$XSAPPNAME.Read"
+      ]
+    },
+    {
+      "name"                 : "Modifier",
+      "description"          : "Modify Data",
+      "scope-references"     : [
+        "$XSAPPNAME.Read",
+        "$XSAPPNAME.Write"
+      ]
+    }
+  ],
+  "role-collections": [
```

Comment:
will this work for XSA?

Reply:
Not sure. If XSA also configures XSUAA instance like all the other CF apps do, my expectation is: YES. This is a feature of xsuaa provisioning.

Reply:
keep that as discussed